### PR TITLE
Issue #6532: aligned javadoc/xdoc for EmptyCatchBlock

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6532 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java [hunk 1]
```diff
@@ -28,100 +28,127 @@
...
+ * <p>
+ * Such empty block would be suppressed:
  * </p>
  * <pre>
- * {@code
  * try {
- *     throw new RuntimeException();
+ *   throw new RuntimeException();
  * } catch (RuntimeException ex) {
  *     //This is expected
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java]: Please align with the `throw` above.

File: src/xdocs/config_blocks.xml [hunk 2]
```diff
@@ -348,21 +348,36 @@ switch (a) {
...
+            </td>
             <td><a href="property_types.html#regexp">Regular Expression</a></td>
-            <td>&quot;.*&quot;</td>
+            <td><code>&quot;.*&quot;</code></td>
             <td>6.4</td>
           </tr>
         </table>
       </subsection>
 
+      <subsection name="Notes" id="EmptyCatchBlock_Notes">
```
Comment (rnveach) [inline src/xdocs/config_blocks.xml]: This section looks out of place when the properties are already described first. By the time we get here, we already know about the properties and what they do.

Comment (pbludov) [reply src/xdocs/config_blocks.xml]: done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java (hunk 1 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java]: done

File: src/xdocs/config_blocks.xml [hunk 3]
```diff
@@ -329,15 +329,23 @@ switch (a) {
...
-      <subsection name="Properties" id="EmptyCatchBlock_Properties">
+      <subsection name="Notes" id="EmptyCatchBlock_Notes">
+        <p>
+          There are two options to make validation more precise: <b>exceptionVariableName</b> and
+          <b>commentFormat</b>.
+          If both options are specified - they are applied by <b>any of them is matching</b>.
+        </p>
+      </subsection>
+
+        <subsection name="Properties" id="EmptyCatchBlock_Properties">
```
Comment (rnveach) [inline src/xdocs/config_blocks.xml]: You indented this line too much.

Comment (pbludov) [reply src/xdocs/config_blocks.xml]: done

**Pull request merged**